Makefile

```diff
@@ -94,14 +81,22 @@ install: node_modules
 run: welcome githooks install build
 	@$(NODE) build/bundle-$(CALYPSO_ENV).js
 
-node-version:
-	@$(BIN)/check-node-version
-
 # a helper rule to ensure that a specific module is installed,
 # without relying on a generic `npm install` command
 node_modules/%: | node-version
 	@$(NPM) install $(notdir $@)
 
+# but `npm install semver` can't rely on node-version because node-version
```

Comment:
The rule directly above this one, `node_modules/%`, is meant to make the rule below implicit. I see that it depends on `node-version` but I'm not sure why (and also not sure what the `|` character does at the beginning of the file list in said rule).